slam-libraries/viam-cartographer/src/main.cc

```diff
@@ -22,12 +24,17 @@ int main(int argc, char** argv) {
 
     sigaction(SIGINT, &sigIntHandler, NULL);
 
+    viam::SLAMServiceImpl slamService;
     int err =
-        viam::slam_service::config::ParseAndValidateConfigParams(argc, argv);
+        viam::config::ParseAndValidateConfigParams(argc, argv, slamService);
     if (err != 0) {
         return err;
     }
 
```

Comment:
Do you want to start up the gRPC server, like we do here: https://github.com/viamrobotics/slam/blob/457f1c3cd997b10e3f3c262a00a21d0f87a3d70d/slam-libraries/viam-orb-slam3/orbslam_server_v1_main.cc#L47-L58? Or will that be in a later ticket?